src/main/java/ApplicationDriver.java

```diff
@@ -1,7 +1,10 @@
 package main.java;
 
-import main.java.controller.MainController;
+import main.java.entity.TodoList;
+import main.java.interface_adapters.MainController;
 import main.java.entity.Task;
+import main.java.use_case.TaskInfo;
```

Comment:
I was just wondering what the reasoning behind moving all the getters into TaskInfo is?